app/utils/session.server.ts

```diff
@@ -0,0 +1,60 @@
+import jwtDecode from 'jwt-decode'
```

Comment:
remove "jwt-decode" and use "jsonwebtoken" instead
"jwt-decode" is a utility library for front-end to check decoded public information only, if you are on server you have to use "jsonwebtoken" with private key so that you make sure it is your generated token